bin/test.bash

```diff
@@ -31,6 +36,15 @@ function sc_help() {
     printf "\t%-23s %s\n" "opensearch" "Open search checks"
     printf "\t%-23s %s\n" "fluentd" "Fluentd checks"
     printf "\t%-23s %s\n" "opensearch-dashboards" "Opensearch dashboards checks"
+    printf "\t%-23s %s\n" "cert-manager" "Cert Manager checks"
+    printf "%s\n" "[NOTE] If no target is specified, the tests will go through all of them."
```

Comment:
Not really true if I understand it correctly. Since if you don't set TARGET it will run the old function `test_apps_sc` and not any of the new ones. And if you specify it it will run the selected new one.
Maybe you could change so that everyone was run if you didn't specify it and then you can add a new entry here like `basic-checks` or something that you can select to run the old function (`test_apps_sc`)

Reply:
Yes  you're right, seems like I forgot it. 
Sure, I will do that

Reply:
I just remembered that you can run all checks of a target like so : test sc opensearch
So I think to keep things like usual, I've changed the message so if no target is specified, it will behave like it is used to, and will run the default apps test